src/Lib/Directory.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Of course you could use BS8.empty instead :)
On Sep 25, 2015 2:25 PM, "Noam Lewis" <lenoam@gmail.com> wrote:

> You need it for "" in `go ""`
> On Sep 25, 2015 2:23 PM, "Eyal Lotem" <notifications@github.com> wrote:
>
>> In src/Lib/Directory.hs
>> <https://github.com/ElastiLotem/buildsome/pull/52#discussion_r40420279>:
>>
>> > @@ -1,3 +1,5 @@
>> > +{-# LANGUAGE BangPatterns #-}
>> > +{-# LANGUAGE OverloadedStrings #-}
>>
>> I don't think you need the overloaded strings. Does hlint not warn?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ElastiLotem/buildsome/pull/52/files#r40420279>.
>>
>